deployment/helm/posix-mapper/values.yaml

```diff
@@ -33,17 +38,24 @@ deployment:
     # - containerPort: 5555
     #   protocol: TCP
 
-    # Resources provided to the Skaha service.
+    # -- Resources provided to the Skaha service.
     resources:
       requests:
+        # -- Memory Requested
         memory: "1Gi"
+        # -- CPU Requested
         cpu: "500m"
       limits:
+        # -- Memory Limit
         memory: "1Gi"
+        # -- CPU Limit
         cpu: "500m"
 
+    # -- Minimum UID
     minUID: 10000
+    # -- Minimum GID
     minGID: 900000
+    # -- Registry URL
     registryURL: https://spsrc27.iaa.csic.es/reg
```

Comment:
This should be relevant only to the deployer, so please replace spsrc27.iaa.csic.es with example.org.